### ISSUE TITLE
in some npm installation scenarios, install fails due to issue with babel 7.8.3

### ISSUE THREAD

**Issue by samueljseay** [Automattic/puppeteer-utils#8]
I am trying to install pupeteer-utils with npm 7.10.0 and the postinstall fails with an error in babel:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /Users/samseay/.npm/_cacache/tmp/git-clone-2ab5e822/node_modules/@babel/helper-compilation-targets/package.json
```
This issue was unique to babel 7.8.3. See https://github.com/babel/babel/issues/11216

A simple fix would be to update babel. Please let me know if I can submit a PR to fix this.

**Comment by WraithKenny** [comment]
This is blocking the https://github.com/woocommerce/woocommerce-admin repo from installing via `npm i`. Maybe this can be a higher priority?

**Comment by brbrr** [comment]
I don't think this project is really maintained. So there is little chance that someone would fix that issue for you. Feel free to open up the PR. 

@juliaamosova FYI

I assume only Woo is currently using it, so maybe it would make sense for yall to keep an eye on this repo?

**Comment by juliaamosova** [comment]
>I assume only Woo is currently using it, so maybe it would make sense for yall to keep an eye on this repo?

Thanks for the ping here @brbrr! We chatted about this with our team and decided that since we are the only ones using `@automattic/puppeteer-utils`, we are going to move all the necessary utilities from this package to `@woocommerce/e2e-utils`: https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e/utils to keep things in order and easier to maintain. We will then archive `@automattic/puppeteer-utils` repo. 

We will be making a post about this as well prior to proceeding to make sure everyone is on the same page. We already have an issue created for it, however: https://github.com/woocommerce/woocommerce-quality/issues/67. cc @bsessions85 here for any comments or feedback. 

@samueljseay we are going to then fix the issue you are having on the `@woocommerce/e2e-utils` side of things after we move to `@woocommerce/e2e-utils`.
